### PR TITLE
refactor(plugin): rename kmov spells to kubemove

### PR DESCRIPTION
**Pull request by ashishranjan738**
Signed-off-by: ashishranjan738 <ashishranjan738@gmail.com>

This commit renames kmov spells with kubemove and adds import alias name.

### CODE REVIEW

**Comment by ashishranjan738** [comment]
@AmitKumarDas please review this.

File: pkg/plugin/v1alpha1/kubemove.go [hunk 1]
```diff
@@ -5,9 +5,9 @@ import "github.com/spf13/cobra"
 // NewCmdKubeMove returns kubemove root command
 func NewCmdKubeMove() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "kubectl kubemove",
-		Short: "kubemove cli-plugin is used for interacting with kmov operators",
-		Long:  "kubemove cli-plugin is used for interacting with kmov operators",
+		Use:   "kubemove",
+		Short: "kubemove is used for interacting with kmov operators",
```
Comment (AmitKumarDas) [inline pkg/plugin/v1alpha1/kubemove.go:9]: There is nothing called `kmov` right? However the message talks about `kmov`

File: pkg/plugin/v1alpha1/kubemove.go [hunk 2]
```diff
@@ -5,9 +5,9 @@ import "github.com/spf13/cobra"
...
+		Long:  "kubemove is used for interacting with kmov operators",
```
Comment (AmitKumarDas) [inline pkg/plugin/v1alpha1/kubemove.go:10]: same comment

**Review by AmitKumarDas — CHANGES_REQUESTED** [review]

**Pull request merged**